doc_indexer/.vscode/launch.json

```diff
@@ -11,6 +11,33 @@
       "program": "${workspaceFolder}/src/main.py",
       "console": "integratedTerminal",
       "args": ["index"]
+    },
```

Comment:
IMO we should ignore .vscode directory from git.

Reply:
I really thought about it. However, I really wanted to have it so that whowever uses VS code can just have the run config to debug w/o effort of configuring it. 
I am currently using VS Code most of the time.